src/sections/Footer.tsx

```diff
@@ -1,20 +1,39 @@
 import React from "react";
 import Image from "next/image";
+import snowman from "~/assets/footer/snowman.png";
+import Link from "next/link";
+import dynamic from "next/dynamic";
+
+const Slider = dynamic(() => import("~/components/PfpSlider"), { ssr: false });
 
 const Footer = () => {
   return (
-    <footer className="grid min-h-screen grid-cols-1 grid-rows-1 bg-red-500">
-      <div className="grid-col relative col-start-1 row-start-1 overflow-clip object-cover p-16">
-        <Image
-          src={"/footer_low.png"}
-          alt={"Footer Blob Image"}
-          objectFit="cover"
-          width={1920}
-          height={1080}
-        />
-      </div>
-      <div className="z-10 col-start-1 row-start-1">
-        <p>HELLO</p>
+    <footer className="mx-auto flex max-w-screen-2xl flex-col items-center">
+      <Image src={snowman} alt="Snowman footer image" />
+      <p className="pb-4 text-center text-2xl">
+        Made with ♥ from the DeltaHacks Team{" "}
+      </p>
+      <Slider />
+      <div className="mt-32 flex w-full max-w-screen-lg flex-col gap-3 py-12 text-center text-xl lg:flex-row lg:gap-8 lg:border-t lg:border-black">
+        <Link
+          href="https://github.com/DeltaHacks/deltahacks-website-2024"
+          className="hover:underline"
+        >
```

Comment:
This is a flexbox, this should be using gap for spacing instead of margins from space-x-8.

Reply:
I fixed it in the commit directly before your review :)